PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cfi.py

```diff
@@ -48,7 +48,19 @@
         cms.InputTag("pfCombinedMVAV2BJetTags"),
         # CTagging
         cms.InputTag('pfCombinedCvsLJetTags'),
-        cms.InputTag('pfCombinedCvsBJetTags')
+        cms.InputTag('pfCombinedCvsBJetTags'),
+        # DeepFlavour
+        # cms.InputTag('pfDeepCSVJetTags:probb'),
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

[I can guess that we need to wait for a reference release inputs to be able to pass the PAT unit tests]